package/pigpio/Config.in

```diff
@@ -0,0 +1,13 @@
+config BR2_PACKAGE_PIGPIO
+	bool "pigpio"
+  depends on BR2_USE_MMU # fork()
+  depends on BR2_TOOLCHAIN_HAS_THREADS
+  depends on !BR2_STATIC_LIBS
+
+	help
+	  PIGPIO daemon.
```

Comment:
Could you change the help to "pigpio is a C library for the Raspberry Pi which allows control of the General Purpose Input Outputs (GPIO)."? Or if there's a better blurb on their website, use that.